index.bs

```diff
@@ -66,16 +66,20 @@ The appropriate times to <a spec=HTML>fetch and process the linked resource</a>
 
   1. [=Parse a URL=] given |el|'s <{link/href}> attribute, relative to |el|'s [=Node/node document=]. If that fails, then return. Otherwise, let |url| be the [=resulting URL record=].
 
-  1. Let |referrer policy| be  the current state of |el|'s <{link/referrerpolicy}> attribute.
+  1. Let |referrerPolicy| be  the current state of |el|'s <{link/referrerpolicy}> attribute.
 
-  1. [=Create a prerendering browsing context=] with |url|, |referrer policy|, and |el|'s [=Node/node document=].
+  1. If |url| is not [=same origin=] with [=Node/node document=]'s [=Document/url=], then:
+
+    1. If the [=list of sufficiently-strict referrer policies=] does not [=list/contain=] |referrerPolicy|, then return.
+
+  1. [=Create a prerendering browsing context=] with |url|, |referrerPolicy|, and |el|'s [=Node/node document=].
 </div>
 
 A user agent must not <a spec=HTML>delay the load event</a> of the <{link}> element's [=Node/node document=] for this link type.
 
 <p class="note">Note that this link type does not fire {{HTMLElement/error}} or {{HTMLElement/load}} events, unlike many other link types that create <a spec=HTML>external resource links</a>.
 
-<p class="issue">TODO: We should define a list of allowable referrer policies to prerender with, and early-return if the above referrer policy is not in this list.
+The <dfn>list of sufficiently-strict referrer policies</dfn> is a [=list=] containing the following subset of [=referrer policies=]: the empty string, "`strict-origin-when-cross-origin`", "`strict-origin`", "`same-origin`", "`no-referrer`".
```

Comment:
You might be able to cross-link these, although getting them to be in code font is harder... I'm pretty sure `<a><code>strict-origin-when-cross-origin</code></a>` will work but I'm not sure about using `[= =]` or backtick shorthand.

Reply:
Hmm, actually I've gone ahead and renamed this the `list of sufficiently-strict speculative navigation referrer policies`, which is a mouthful, but I think it gives us good "coverage" for future processing models that use this infra later.